src/pages/workspace/accounting/WorkspaceAccountingPage.tsx

```diff
@@ -231,7 +226,7 @@ function PolicyAccountingPage() {
                             </View>
                         )}
                         <MenuItemList
-                            menuItems={menuItems}
+                            menuItems={!policyIsConnectedToAccountingSystem ? connectionsMenuItems : qboConnectionMenuItems}
```

Comment:
```suggestion
                            menuItems={policyIsConnectedToAccountingSystem ? qboConnectionMenuItems : connectionsMenuItems}
```
Avoid negative conditions when possible